examples/weathertabs/js/actions/WeatherActions.js

```diff
@@ -0,0 +1,33 @@
+var $ = require('jquery');
+
+class WeatherActions {
+  constructor() {
+    this.generateActions(
+      'setLoading',
+      'setWeather',
+      'showRaw'
+    );
+  }
+
+  loadWeather(location) {
+    this.dispatch(location);
+    $.ajax('http://api.openweathermap.org/data/2.5/weather', {
```

Comment:
@emroot I tried using the fetch lib but it did not work with commonJS, their npm module does not properly do module.exports